bundles/org.openhab.binding.pulseaudio/README.md

```diff
@@ -38,10 +38,13 @@ binding.pulseaudio:sourceOutput=false
 
 ## Thing Configuration
 
-The Pulseaudio bridge requires the host (ip address or a hostname) and a port (default: 4712) as a configuration value in order for the binding to know where to access it.
-A Pulseaudio device requires at least an identifier. For sinks and sources, you can use the name or the description. For sink inputs and source outputs, you can use the name or the application name.
+The Pulseaudio bridge requires the host (ip address or a hostname) and a port (default: 4712) as a configuration value in order for the binding to know where to access it. A Pulseaudio device requires at least an identifier. For sinks and sources, you can use the name or the description. For sink inputs and source outputs, you can use the name or the application name.
```

Comment:
It seems like nothing is really changed here rendering-wise, but putting two sentences on the same line seems like the wrong direction, so it would be better to split them into separate lines:
```suggestion
The Pulseaudio bridge requires the host (ip address or a hostname) and a port (default: 4712) as a configuration value in order for the binding to know where to access it.
A Pulseaudio device requires at least an identifier.
For sinks and sources, you can use the name or the description.
For sink inputs and source outputs, you can use the name or the application name.
```